# action/PatrolAction.action
# Goal
duration patrol_duration
---
# Result
geometry_msgs/Pose[] people_positions
geometry_msgs/Pose[] cat_positions
geometry_msgs/Pose[] dog_positions
---
# Feedback
int32 new_people
int32 new_cats
int32 new_dogs
geometry_msgs/Pose last_detection_pose